pkg/internalapis/cnsvolumeinfo/v1alpha1/cnsvolumeinfo_types.go

```diff
@@ -28,6 +28,12 @@ type CNSVolumeInfoSpec struct {
 
 	// vCenterServer is the IP/FQDN of the vCenter host on which the CNS volume is accessible.
 	VCenterServer string `json:"vCenterServer"`
+
+	// ID of the storage policy
+	StoragePolicyId string `json:"storagepolicyid,omitempty"`
+
+	// Name of the storage policy
+	StoragePolicyName string `json:"storagepolicyname,omitempty"`
```

Comment:
we do not need to storestorage policy name. policy name can be changed, after it is created.

Reply:
Can you make the json name camel cased?